clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -48,17 +48,22 @@
     public static final String MAX_POLL_RECORDS_CONFIG = "max.poll.records";
     private static final String MAX_POLL_RECORDS_DOC = "The maximum number of records returned in a single call to poll().";
 
+    /** <code>max.poll.interval.ms</code> */
+    public static final String MAX_POLL_INTERVAL_MS_CONFIG = "max.poll.interval.ms";
```

Comment:
As discussed offline, `max.poll.interval.ms` probably won't make any sense wrt workers. Something like `rebalance.timeout.ms` does and we can follow up with necessary KIP changes on the mailing list.